src/p2p/net_node.inl

```diff
@@ -1957,6 +1983,52 @@ namespace nodetool
   }
   //-----------------------------------------------------------------------------------
   template<class t_payload_net_handler>
+  epee::net_utils::zone node_server<t_payload_net_handler>::send_txs(std::vector<cryptonote::blobdata> txs, const epee::net_utils::zone origin, const boost::uuids::uuid& source, const bool pad_txs)
+  {
+    namespace enet = epee::net_utils;
+
+    const auto send = [&txs, &source, pad_txs] (std::pair<const enet::zone, network_zone>& network)
+    {
+      if (network.second.m_notifier.send_txs(std::move(txs), source, pad_txs))
```

Comment:
This should be `pad_txs || network.first != enet::zone::public_` to match existing behavior. That way, if the noise feature is disabled on the network, padding is still automatically used.

Will push a change later (or tomorrow).